alternator/executor.cc

```diff
@@ -128,6 +128,20 @@ void executor::supplement_table_info(rjson::value& descr, const schema& schema,
 // See https://github.com/scylladb/scylla/issues/4480
 static constexpr int max_table_name_length = 222;
 
+static bool valid_table_name_chars(std::string_view name) {
+    for(auto c : name) {
```

Comment:
Coding style: space after `for`, it isn't a function.

Reply:
added